redfish-core/lib/assembly.hpp

```diff
@@ -540,6 +620,39 @@ inline void setAssemblylocationIndicators(
                         "xyz.openbmc_project.adcsensor.service", "replace");
                 }
             }
+            // Special handling for LCD and base panel CM.
```

Comment:
Can you explain more **why** we are doing this?

Reply:
I assume you are asking to add more comment in the code here. Though it is pointed that it is required for CM functionality of panel. 
Is there something specific you want me to add?

Reply:
I asked for a comment below so as long as you do that I am good

Reply:
Done